data/authorities.json

```diff
@@ -829,9 +829,6 @@
       "nv-mt-wheeler-power": {
         "name": "Mt Wheeler Power"
       },
-      "nv-nevada-power": {
```

Comment:
Sierra Pacific Power didn't get removed because there's still 1 row in the spreadsheet that references `Sierra Pacific Power Co [Berkshire Hathaway Inc]` in NV with a NV zip code. I don't think we can override this one because unfortunately that utility also exists in California with the same name and utility code, and the CA utility is actually legitimate:
<img width="679" alt="Screenshot 2024-03-20 at 1 23 26 PM" src="https://github.com/rewiringamerica/api.rewiringamerica.org/assets/17071438/9bbd57c0-7b2f-48f0-955b-2b976aec12d4">

functionally this shouldn't matter since it only applies to 1 zip code, 89803, and the primary utility for that zip code is Mt Wheeler power so it wouldn't show up in the dropdown anyway. it's just a little bit weird that Sierra Pacific still shows up in authorities.json

Reply:
That kinda feels to me like an error in the dataset. Sometimes you see cases like this when the ZIP is right on a state border (e.g. a little piece of VA on the KY border is served by Kentucky Utilities), but 89803 isn't anywhere near the CA/NV border. I agree it's OK to let this slide.